src/utils/licenses.js

```diff
@@ -12,7 +12,7 @@ export const LICENSES = {
       NC: false,
       ND: false,
     },
-    FULL: 'CC0 1.0 Universal',
+    FULL: 'Creative Commons Zero 1.0 Universal',
```

Comment:
I believe in the case of `CC0` that its long name is still `CCO 1.0 Universal`, and that it is not spelled out in this fashion. 

Please see the license deed itself to verify: https://creativecommons.org/publicdomain/zero/1.0/

Reply:
According to the website, it is indeed CC0.

Reply:
@thisisharsh7 Great! As soon as you adjust this I can move forward with approval :)